pyproject.toml

```diff
@@ -56,6 +56,8 @@ tox = "^3.24"
 click = "^8.1"
 Mako = "^1.2"
 deprecation = "^2.1"
+ni-python-styleguide = "^0.3"
```

Comment:
The current version of ni-python-styleguide is 0.4: https://github.com/ni/python-styleguide/releases